test/lib/buildFactory.test.js

```diff
@@ -116,12 +117,16 @@ describe('Build Factory', () => {
         const eventId = '12345f642bbfd1886623964b4cff12db59869e5d';
         const sha = 'ccc49349d3cffbd12ea9e3d41521480b4aa5de5f';
         const scmUri = 'github.com:12345:master';
+        const scmRepo = {
+            name: 'screwdriver-cd/models'
+        };
+        const prRef = 'pull/3/merge';
         const username = 'i_made_the_request';
         const dateNow = Date.now();
         const isoTime = (new Date(dateNow)).toISOString();
         const container = 'node:4';
         const steps = [
-            { name: 'sd-setup' },
+            { command: 'git clone', name: 'checkout-code' },
```

Comment:
Makes sense to add `sd-` as the prefix! 👍 Keep the `sd-setup` for now and will see if we can move those commands out in another PR. Rename this to  `sd-checkout-code`. @tkyi We need to change scm-base, scm-bitbucket and scm-github with the step name  `sd-checkout-code`.